### DISCUSSION TITLE
Tacotron2 GST on Blizzard not converging

### DISCUSSION THREAD

**Discussion by manmaynakhashi** [coqui-ai/TTS#1935] [General Q&A] [answered]
![al-0-s](https://user-images.githubusercontent.com/108130689/188438695-e8b48f54-9de1-406f-9282-2478fa60077f.png)
![al-0](https://user-images.githubusercontent.com/108130689/188438708-41498d5f-b9e1-425a-b8a4-df8d44c020fd.png)
![al-1-s](https://user-images.githubusercontent.com/108130689/188438717-559d9b8e-d2e8-475c-ba29-5072ed633054.png)
![al-1](https://user-images.githubusercontent.com/108130689/188438722-c57e2663-c4d8-4ffc-b003-3b1a3ddd7451.png)

alignments are not improving further

[config.txt](https://github.com/coqui-ai/TTS/files/9489370/config.txt)

**Comment by manmaynakhashi** [comment, marked as answer]
@WeberJulian can you check my config and check if something is wrong.

**Reply by WeberJulian** [reply]
I see that you are using `"ga_alpha": 0.0,` please use the latest recipe on the dev branch, training is more stable there.

**Reply by manmaynakhashi** [reply]
@WeberJulian sure i'll try this and update the results

**Reply by manmaynakhashi** [reply]
@WeberJulian i can see "ga_alpha": 0.0, in dev branch ljspeech recipe

**Reply by WeberJulian** [reply]
https://github.com/coqui-ai/TTS/blob/dev/recipes/blizzard2013/tacotron2-Capacitron/train_capacitron_t2.py

**Comment by KyuubiYoru** [comment]
How many training steps?

**Reply by manmaynakhashi** [reply]
@KyuubiYoru 400k

**Reply by KyuubiYoru** [reply]
On my experiments with tacotron2 the alignment was clearly visible at around 10k steps, the end result was also lacking but I can share my config after I get home in 10h.
A lot of settings are different in my config.

**Reply by manmaynakhashi** [reply]
@KyuubiYoru sure , thank you for your support.

**Reply by KyuubiYoru** [reply]
This is the last config I used.
[config.txt](https://github.com/coqui-ai/TTS/files/9509795/config.txt)
I wasn't happy with the quality and I think there might be an overfitting problem. Maybe you can improve things with this configuration.

Results looked like this:
![image](https://user-images.githubusercontent.com/33806787/188973243-cd301c1a-1647-4918-af4f-0649cb1a9ca8.png)

![image](https://user-images.githubusercontent.com/33806787/188973385-d1b00168-6f0f-401a-8700-77f9eaff2e9e.png)

**Reply by manmaynakhashi** [reply]
@KyuubiYoru thank you i'll post updated config once i get a good output
